easybuild/easyblocks/boostcray.py

```diff
@@ -1,222 +0,0 @@
-##
```

Comment:
Can you please confirm that you really need to remove this easyblock for TSA and that we don't need anymore on Daint?

Reply:
It is a really outdated easyblock, we can now safely rely on the default easyblock for Boost